src/ToDo.UI/App.xaml.host.cs

```diff
@@ -20,6 +17,16 @@ private static IHost BuildAppHost()
 		var useMocks = false;
 #endif
 
+#if __WASM__
+		var stringUri = WebAssemblyRuntime.InvokeJS("window.location.href;");
+		var query = new Uri(stringUri).Query;
+		var queriesValues = System.Web.HttpUtility.ParseQueryString(query);
+
+		if (queriesValues.TryGetValue(_mock, out var value) && bool.TryParse(value, out var isMocked))
```

Comment:
```suggestion
		if (queriesValues["mock"] is string mock && mock?.ToLower() == "true")
```
or better use mock=1, and just compare "1" here without tolower